test/spoof/spoof.go

```diff
@@ -75,6 +74,8 @@ var _ Interface = (*SpoofingClient)(nil)
 // https://github.com/kubernetes/apimachinery/blob/cf7ae2f57dabc02a3d215f15ca61ae1446f3be8f/pkg/util/wait/wait.go#L172
 type ResponseChecker func(resp *Response) (done bool, err error)
 
+type FormatLogger func(template string, args ...interface{})
```

Comment:
I am not thrilled that we eliminate non formatting loggers, but for tests it probably doesn't matter much.

Reply:
Yeah, if the methods were called the same things on `logger` and `testing.T` I would have defined an interface, but it's `Info[f]` and `Log[f]` respectively :(